virtcontainers/pkg/types/pcipath.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2020 David Gibson, Red Hat Inc.
```

Comment:
Just wondering if this needs your name here (and in the tests file)? All the other copyright headers just specify the date and name for contributors working for a company.

Reply:
I don't think it's necessary, but it is the convention I've been used to.

But.. I checked our internal guidelines for this, and it looks like I'm a bit out of date, so I've updated accordingly.

Reply:
Well.. mostly.  Our guidelines recommend against including "(c)" and year info, but checkpatch objects to their absence.

Reply:
Thanks @dgibson.